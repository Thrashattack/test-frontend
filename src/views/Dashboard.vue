<template>
  <div class="Dashboard d-flex flex-column align-items-center justify-content-center">
    <router-link
      tag="div"
      path="Lista de empresas"
      :to="{ name: 'empresas.list' }"
      class="__Link d-flex flex-column justify-content-center align-items-center"
    >
    Empresas
    </router-link>
  </div>
</template>

<script>
export default {
  name: "Dashboard"
}
</script>

<style lang="scss" scoped>
.Dashboard {
  min-height: 100%;
  min-width: 100%;
  background: #F7F7F7;
  .__Link {
    cursor: pointer;
    padding: 10px 70px;
    box-shadow: #acacac 1px 1px 5px;
    background: #FFFFFF;
    border: 1px solid #ECECEC;
    border-radius: 3px;
    box-sizing: border-box;
  }
}
</style>
