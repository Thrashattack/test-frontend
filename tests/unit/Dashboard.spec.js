import { shallowMount } from "@vue/test-utils";
import Dashboard from "@/views/Dashboard.vue";

describe("Dashboard.vue", () => {
  it("renders the view", () => {
    const wrapper = shallowMount(Dashboard);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
