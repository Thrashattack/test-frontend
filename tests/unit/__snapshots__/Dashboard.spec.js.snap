// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard.vue renders the view 1`] = `<div class="Dashboard"></div>`;
